languages/csharp/exercises/concept/inheritance/.docs/after.md

```diff
@@ -0,0 +1,71 @@
+In C#, a _class_ hierarchy can be defined using _inheritance_, which allows a derived class (`Car`) to inherit the behavior and data of its parent (general) class (`Vehicle`). If no parent is specified, the class inherits from the `object` class. To prevent a class being inherited, add the [`sealed` modifier][sealed-classes].
```

Comment:
```suggestion
In C#, a _class_ hierarchy can be defined using _inheritance_. This allows a derived class (`Car`) to inherit the behavior and data of its parent class (`Vehicle`). If no parent is specified, the class inherits from the `object` class. To prevent a class being inherited, add the [`sealed` modifier][sealed-classes].
```